CGE-mlst/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM debian:stretch
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="mlst"
+LABEL software.version="2.19.0"
```

Comment:
I think this version should be 2.0.4? That's the latest tag I can see on the bitbucket repo